conda_build/utils.py

```diff
@@ -1504,36 +1504,7 @@ def filter_info_files(files_list, prefix):
 
 
 def rm_rf(path, config=None):
-    if conda_46:
-        return _rm_rf(path)
-    if os.path.isdir(path):
-        try:
-            # subprocessing to delete large folders can be quite a bit faster
-            if on_win:
-                subprocess.check_call(f'rd /s /q {path}', shell=True)
-            else:
-                try:
-                    os.makedirs('.empty')
-                except:
-                    pass
-                del_dir_cmd = 'rsync -a --delete .empty {}/'
-                subprocess.check_call(del_dir_cmd.format(path).split())
-                try:
-                    shutil.rmtree('.empty')
-                except:
-                    pass
-        # we don't really care about errors that much. People can and should
-        #     clean out their folders once in a while with "purge"
-        except:
-            pass
-
-    conda_log_level = logging.WARN
-    if config and config.debug:
-        conda_log_level = logging.DEBUG
-    with LoggingContext(conda_log_level):
-        # this clears out the path from conda's cache, which otherwise thinks
-        #    that things are still installed here
-        _rm_rf(path)
+    return _rm_rf(path)
```

Comment:
Should remove this indirection and remove the `as _rm_rf` from the `import` line.

Reply:
I agree, unfortunately doing so will deprecate the `config` argument

I wanted to get https://github.com/conda/conda/pull/12125 merged before starting to mark  deprecations per [CEP9](https://github.com/conda-incubator/ceps/blob/main/cep-9.md)

Reply:
Ah, right. In the moment I forgot that we have a few downstream users (including code I wrote :facepalm:) that import things from everywhere.